### ISSUE TITLE
[Feature]: Creating notes from cards should place them in the same folder as the board

### ISSUE THREAD

**Issue by cherti** [mgmeyers/obsidian-kanban#719]
### Goal or desired outcome of this feature

Currently, when you create a note from a Card, this note can be placed either in the default folder of the vault or in a specific, dedicated folder for all boards. This severely limits the usability of having multiple kanban-boards in one vault for different purposes (for example project-specific), because their notes would get mixed up (unless manually sorted away).
If notes could be created in the same folder as the board, this would alleviate this issue.

### Describe the feature

The config option "Note Folder" should have an entry such as "same as board", which would cause note files to be created in the same folder as the board they are created from, keeping things orderly and tidy when more than one board is used in a vault.

### Can you think of any alternatives or work-arounds?

Currently I only see manually sorting note cards.

**Comment by yakschuss** [comment]
+1

**Comment by SimonHryszko** [comment]
Agree, I would love to see that default is `sameDir` or `sameDir/name` I think that would make more sense

**Comment by RomanMunar** [comment]
You can just set the `Default location for new notes` to `Same folder as the current` in `Settings > Files and links`.

**Comment by Yehuda-Bergstein** [comment]
> You can just set the `Default location for new notes` to `Same folder as the current` in `Settings > Files and links`.

The problem I believe is that this is for all boards, not on her per board basis.
Would be neat to have an option to set multiple locations for each board
